scripts/export.sh

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env bash
 #
-# Usage: ./export.sh [SNAPSHOT_FILE] [EXPORT_DIR]
+# Usage: ./export.sh [SNAPSHOT_FILE] [EXPORT_DIR] [TASKS]
```

Comment:
We can now pass a task and will produce only the CSV for that task.

Reply:
Can you add a switch case here to add validation for task? Bc some tasks do not exist in certain epochs or have different names

Reply:
I thought about that but since Filet is thought to be used with all tasks enabled I don't think is worth adding any checks. The only times it'll be run with specific tasks are when we want to run things manually. If the task doesn't exist, we'll get errors.

Reply:
It will error though even if you have all tasks enabled if you use a later version of lily on earlier parts of the chain. For e.g., `id_addresses` amongst other tasks.

Reply:
Just to double check, if I run `lily job run walk --from 0 --to 1000` with the current version of Lily, will it fail because tasks have changed?

Reply:
A better question is, what would be a snapshot in which `lily` errors when walking with all tasks enabled?